docs/detections/visual-event-analyzer.asciidoc

```diff
@@ -40,6 +49,20 @@ You can also analyze events from <<timelines-ui,Timelines>>
 
 Inside the visual analyzer, each cube represents a process (i.e. an executable file or network event). Click and drag in timeline view to see all process relationships. 
 
+To help understand what field were used to create the process, select the **Process Tree** to view the schema that created the graphical view. The fields included are:
+
+-`SOURCE`: Can be either `endpoint` or `winlogbeat`.
+- `ID`: What KQL entity created the process.
```

Comment:
hmm _KQL_ is a query language, maybe this instead: **`ID`: The field from events (or maybe documents?) used to uniquely identify a node**